util.py

```diff
@@ -1967,7 +1969,32 @@ def parse_mf2(input, url=None, id=None):
   return mf2py.parse(url=url, doc=input, img_with_alt=True)
 
 
-def fetch_mf2(url, get_fn=requests_get, gateway=False, **kwargs):
+def parse_http_equiv(input, **kwargs):
+  """Parses http_equiv meta tag, if available.
+
+  Args:
+    input: unicode HTML string, :class:`bs4.BeautifulSoup`, or
+      :class:`requests.Response`
+
+  Returns: str, empty if not available or a url if available
+  """
+  if not isinstance(input, (bs4.BeautifulSoup, bs4.Tag)):
+    input = parse_html(input)
+
+  element = input.find("meta", attrs={'http-equiv': 'refresh'})
```

Comment:
very minor style nit, mind switching to single quotes?

Reply:
fixed!